### PR TITLE
OSS/open source -> Redis Community Edition

### PR DESCRIPTION
**Pull request by dwdougherty**
Change most occurrences of OSS or open source to Redis Community Edition (or CE). References to UI elements have been left untouched until the software is modified.

### CODE REVIEW

**Review by rrelledge — COMMENTED** [review]
The r7 and r7.2 breaking changes embeds are used in release notes and the Cloud changelog. I'm not sure if we should change these or leave them as is.

**Comment by dwdougherty** [comment]
I was wondering about that myself. I'll do whatever you think is best, @rrelledge .

**Comment by rrelledge** [comment]
Since 7.0 and 7.2 predate CE and these are only used in release notes and the changelog, I think it makes sense to leave them as OSS, but I don't know if it's worth changing them back.
@mich-elle-luna, what are your thoughts on this? Should we just keep the CE changes for these embeds or change them back?

File: content/get-started/_index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Get started with source available Redis
+description: Get started with Redis Community Edition
```
Comment (rrelledge) [inline content/get-started/_index.md:2]: After taking a closer look at this page on https://redis.io/docs/latest/get-started/, maybe this should just be "Redis" or changed back to "source available Redis". The first half of the doc seems to be mostly related to Redis CE, but the second half has Enterprise links.
```suggestion
description: Get started with source available Redis
```

Comment (dwdougherty) [reply content/get-started/_index.md:2]: I think just "Redis" is probably the right idea here.

File: content/operate/rc/compatibility.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-Title: Redis Cloud compatibility with source available Redis
+Title: Redis Cloud compatibility with Redis Community Edition
 alwaysopen: false
 categories:
 - docs
 - operate
 - rc
-description: Redis Cloud compatibility with source available Redis.
+description: Redis Cloud compatibility with Redis Community Edition.
 linkTitle: Source available compatibility
```
Comment (cmilesb) [inline content/operate/rc/compatibility.md:9]:
```suggestion
linkTitle: Community Edition compatibility
```

**Review by cmilesb — APPROVED** [review]
Just one place - looks like you got the rest of the changes.

File: content/operate/rc/compatibility.md (hunk 2 above)
Comment (dwdougherty) [reply content/operate/rc/compatibility.md:9]: Thank you, @cmilesb ! Can you modify your suggestion to include Redis?

linkTitle: Redis Community Edition compatibility

Comment (cmilesb) [reply content/operate/rc/compatibility.md:9]: @dwdougherty Done!

**Review by mich-elle-luna — APPROVED** [review]
just one typo, thank you!

**Comment by amiramm** [comment]
> Since 7.0 and 7.2 predate CE and these are only used in release notes and the changelog, I think it makes sense to leave them as OSS, but I don't know if it's worth changing them back. @mich-elle-luna, what are your thoughts on this? Should we just keep the CE changes for these embeds or change them back?

These older releases are actually OSS, license-wise, and are expected to forever be.

**Comment by amiramm** [comment]
@mich-elle-luna, can we add a reference to the change we've made in terminology in a central location if we haven't already? Something that explains to users that we're now referencing to OSS as CE? @K-Jo we should later use that to refer to your blog.

**Comment by mich-elle-luna** [comment]
> > Since 7.0 and 7.2 predate CE and these are only used in release notes and the changelog, I think it makes sense to leave them as OSS, but I don't know if it's worth changing them back. @mich-elle-luna, what are your thoughts on this? Should we just keep the CE changes for these embeds or change them back?
> 
> These older releases are actually OSS, license-wise, and are expected to forever be.

Hi, yes, we should change these back to OSS.

File: content/glossary/_index.md [hunk 3]
```diff
@@ -309,7 +309,7 @@ The on-premises version of Redis Enterprise.
 {{%/definition%}}
 
 {{%definition "Redis instance"%}}
-Single-threaded Redis OSS database.
+Single-threaded Redis Community Edition database.
```
Comment (mich-elle-luna) [inline content/glossary/_index.md:312]:
```suggestion
Single-threaded Redis Community Edition database. Redis OSS was renamed Redis Community Edition (CE) with the v7.4 release.
```

File: content/operate/oss_and_stack/_index.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Redis OSS and Stack
-description: Operate Redis OSS and Redis Stack
-linkTitle: Redis OSS and Stack
+title: Redis Community Edition and Stack
+description: Operate Redis Community Edition and Redis Stack
```
Comment (mich-elle-luna) [inline content/operate/oss_and_stack/_index.md:3]:
```suggestion
description: Operate Redis Community Edition and Redis Stack. Redis OSS was renamed Redis Community Edition (CE) with the v7.4 release.
```

**Review by mich-elle-luna — COMMENTED** [review]
Added a line about the renaming to the glossary and landing page as requested by Amiram.

**Comment by rrelledge** [comment]
> > > Since 7.0 and 7.2 predate CE and these are only used in release notes and the changelog, I think it makes sense to leave them as OSS, but I don't know if it's worth changing them back. @mich-elle-luna, what are your thoughts on this? Should we just keep the CE changes for these embeds or change them back?
> > 
> > 
> > These older releases are actually OSS, license-wise, and are expected to forever be.
> 
> Hi, yes, we should change these back to OSS.

Thanks for confirming. We already changed the 7.0 and 7.2 breaking changes embeds back to OSS in the "[Revert changes to Redis 7 and 7.2 breaking changes embeds](https://github.com/redis/docs/pull/305/commits/39df40985ff6e478b5c1987bd292653bac6ff406)" commit.

**Pull request merged**